examples/oio-idws-soap/bootstrap-scenario/pom.xml

```diff
@@ -70,7 +60,7 @@
 		<dependency>
 			<groupId>org.apache.santuario</groupId>
 			<artifactId>xmlsec</artifactId>
-			<version>2.1.4</version>
+			<version>2.0.8</version>
```

Comment:
Du skal op på 2.1.4 ellers får vi en DependentABot fejl.

Reply:
Det vi ikke pga. afhængigheder mellem OIO SAML og OIO IDWS for bootstrap scenariet